e2etest/bgp_test.go

```diff
@@ -149,6 +149,11 @@ var _ = ginkgo.Describe("BGP", func() {
 
 			validateService(cs, svc, epNodes)
 		})
+
+		ginkgo.It("would check that FRR is running in the speaker", func() {
+			paired := frrIsPairedOnPods(cs, frrContainerIPv4)
+			framework.ExpectEqual(paired, true)
```

Comment:
could add the opposite check for non-frr test jobs (make sure the frr container isn't running when it's not supposed to be)